### PR TITLE
Reduce coverage load

### PR DESCRIPTION
**Pull request by wilzbach**
It looks like it's not that difficult to skip, so I quickly did it.

Btw the last few `runnable` tests look worrying on Linux, is this known?
Output is copied from a _passed_ test on AutoTester.

```
./runnable/test16096.sh: line 4: ${output_file}: ambiguous redirect
 ... runnable/test10386.sh
 ... runnable/template2.d           (-O -inline -release)
 ... runnable/gdb15729.sh
 ... runnable/test10567.sh
Missing separate debuginfo for /lib/libgcc_s.so.1
Try: yum --enablerepo='*debug*' install /usr/lib/debug/.build-id/22/ffc1dcfb5f5e40f3331f5aec6eb25b3d8c1e87.debug
RESULT=$1 = 1234
 ... runnable/link14198b.sh
 ... runnable/link14198a.sh
rm: cannot remove ‘generated/runnable/test14198b’: No such file or directory
```

### CODE REVIEW

**Comment by WalterBright** [comment]
In order to not do the link step, the `-c` switch needs to be thrown on the compiler. I don't see that in the diff - can you check, please?

**Comment by wilzbach** [comment]
Oh this PR wasn't really ready,  sorry about not marking it as WIP. I was still trying to figure out how to cover those 133 lines in `inline.d`, but maybe the reduction of the run-time from 19min to 8min is a reasonable trade-off for now ;-)

> In order to not do the link step, the -c switch needs to be thrown on the compiler. I don't see that in the diff - can you check, please?

Sure, I provided a logging option to show this at #6012.

Btw in case you have noticed that the mapping on same files at CodeCov is still wrong (even with the `.lst` files stored in `src`), I reported that and it should be fixed within the next hours :)

File: travis.sh [hunk 1]
```diff
@@ -52,8 +52,13 @@ if [ "${CIRCLECI}" != "true" ] ; then
...
 
+QUICK_BUILD=0
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] ; then
+	QUICK_BUILD=1
+fi
+
 # test fewer compiler argument permutations for PRs to reduce CI load
-if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+if [ $QUICK_BUILD -eq 1 ]; then
     make -j$N -C test MODEL=$MODEL
```
Comment (MartinNowak) [inline travis.sh]: Coverage of the inliner is actually interesting, please add at least `-inline` back.

Comment (MartinNowak) [inline travis.sh]: The rest will become relevant if the crazy people really continue to port the backend.

Comment (wilzbach) [inline travis.sh]: `QUICK_BUILD` is already `0` and it executes ` make -j2 -C test MODEL=32 'ARGS=-O -inline -release'`.
I tried before using the "quick build" option, but it didn't make a significant build time difference.

Comment (WalterBright) [inline travis.sh]: I don't see much point in -release for testing coverage in the compiler.

**Pull request merged**